frontend/src/components/kl-items.js

```diff
@@ -45,6 +52,9 @@ export default {
                 item.totalDps = totalDps(item, newVal).toFixed(2);
                 return item;
             });
+        },
+        sort: function (newVal) {
+            this[newVal]();
         }
```

Comment:
List will become unsorted if you add another item after sorting, won't it?

The proper way is like so:

- Have a value to indicate which way to sort (you already have it, `sort` it is)
- Have a [computed property](https://vuejs.org/v2/guide/computed.html) `sorted` which is, well, a sorted list of items, created using `items` and `sort`
- Bind your list in template to `sorted` instead of `items`